src/main/java/com/cloudbees/jenkins/plugins/bitbucket/endpoints/AbstractBitbucketEndpointDescriptor.java

```diff
@@ -57,9 +58,12 @@ public ListBoxModel doFillCredentialsIdItems(@QueryParameter String serverUrl) {
         result.includeMatchingAs(
                 ACL.SYSTEM,
                 jenkins,
-                StandardUsernameCredentials.class,
+                StandardCredentials.class,
                 URIRequirementBuilder.fromUri(serverUrl).build(),
-                CredentialsMatchers.instanceOf(StandardUsernamePasswordCredentials.class)
+                CredentialsMatchers.anyOf(
```

Comment:
No, you should be using `AuthenticationTokens.matcher(BitbucketAuthenticator.class)` here. No need to test for `instanceOf` any more. In fact this switch is the key reason to move to AuthenticationTokens... as it means an extension plugin can register more authenticator sources and then the support wires right through